### PR TITLE
Add loaders

### PR DESCRIPTION
**Pull request by tylerlan**

### CODE REVIEW

File: src/components/AddNewDestination/AddNewDestinationModal.jsx [hunk 1]
```diff
@@ -10,6 +10,8 @@ class AddNewDestinationModal extends Component {
     this.state = { modalOpen: false };
   }
 
+  // TODO: TEST SIMULATE OPEN AND CLOSE
```
Comment (joshwyatt) [inline src/components/AddNewDestination/AddNewDestinationModal.jsx:13]: @tylerlan these will be exceptionally helpful for the future, thanks.

File: src/actions/index.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import * as TYPES from '../constants/constants';
-import * as HELPERS from '../utils/helpers';
+import * as HELPERS from '../utils/apiHelpers';
```
Comment (joshwyatt) [inline src/actions/index.js:2]: @tylerlan nice change

File: src/components/Destinations/Destination/DestinationHeader.jsx [hunk 3]
```diff
@@ -11,12 +11,12 @@ const DestinationHeader = ({ name }) => (
...
-
 DestinationHeader.propTypes = {
   name: PropTypes.string.isRequired,
 };
 
 DestinationHeader.defaultProps = {
   name: '',
 };
+
+export default DestinationHeader;
```
Comment (joshwyatt) [inline src/components/Destinations/Destination/DestinationHeader.jsx:22]: @tylerlan yes this is better

File: src/utils/journeyHelpers.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
+import walkIcon from '../images/walk-web-med.png';
```
Comment (joshwyatt) [inline src/utils/journeyHelpers.js:1]: @tylerlan good choice moving all this over here

**Review by joshwyatt — COMMENTED** [review]
@tylerlan this is really incredible work, thanks so much!

I'm ready to merge this PR with just 2 small additions.

1) Please edit `.env.develpment` to read `REACT_APP_API_SERVER_URL=http://localhost:8001` in order to hit the port that the dev container is running on after `npm start`. See `docker-compose.yml` for why we know we need to hit `8001`
2) Please add `.env.transit-api` to `.gitignore`. We both expect this file to be mounted into the API container (again see `docker-compose.yml`), and, do not wish to commit it.

Ping me when these 2 small additions are complete, and otherwise this is just great. Thanks for all the tests and for your superlative work on this widget.

**Comment by tylerlan** [comment]
@joshwyatt done.

**Pull request merged**